src/test/scala/org/broadinstitute/dsde/workbench/sam/api/MockSamRoutesBuilder.scala

```diff
@@ -27,6 +27,8 @@ class MockSamRoutesBuilder(allUsersGroup: WorkbenchGroup)(implicit system: Actor
   private val userServiceBuilder: MockUserServiceBuilder = MockUserServiceBuilder()
   private val mockTosServiceBuilder = MockTosServiceBuilder()
 
+  val mockResourceService = mock[ResourceService]
```

Comment:
I didnt want to have to add a whole MockResourceServiceBuilder thing just for this so I made this mock available for mocking in the test i wrote